documentation-website/Writerside/snippets/exposed-dsl/src/main/kotlin/org/example/examples/DeleteExamples.kt

```diff
@@ -41,10 +37,21 @@ class DeleteExamples {
             THEN DELETE
          */
 
+        StarWarsFilmsIntIdTable.insertIgnore {
+            it[sequelId] = MOVIE_SEQUEL_ID
+            it[name] = "The Force Awakens"
+            it[director] = "J.J. Abrams"
+        }
```

Comment:
Minor, because I can't find any use of the above SQL in the documentation:

Just pointing out that the SQL on lines 34-37 above would only be generated by H2.
It doesn't seem like the lines are included anywhere so far, but if this `joinDelete` example is now only used by MySQL, it would generate SQL like this, since MySQL does not rely on `MERGE` statement:
```sql
DELETE actors
FROM star_wars_films_table INNER JOIN actors
ON star_wars_films_table.id = actors.sequel_id
WHERE actors.sequel_id > 2
```
But I think it's ok to use the H2 example in the future since join delete is supported by H2 and the documentation doesn't specify which db is being used, so the default H2 is implied. I get that it is only being run on MySQL because of `insertIgnore`.